src/components/breadcrumbs.tsx

```diff
@@ -0,0 +1,73 @@
+/**
+ * Copyright (c) 2025, RTE (http://www.rte-france.com)
+ * This Source Code Form is subject to the terms of the Mozilla Public
+ * License, v. 2.0. If a copy of the MPL was not distributed with this
+ * file, You can obtain one at http://mozilla.org/MPL/2.0/.
+ */
+
+import * as React from 'react';
+import { Breadcrumbs as MuiBreadcrumbs } from '@mui/material';
+import { ArrowForwardIos, MoreHoriz, RemoveRedEye } from '@mui/icons-material';
+import Tooltip from '@mui/material/Tooltip';
+import { Box, Select } from '@mui/material';
+import { CurrentTreeNode } from './graph/tree-node.type';
+import { useDispatch, useSelector } from 'react-redux';
+import { AppState } from '../redux/reducer';
+import { RootNetworkMetadata } from './graph/menus/network-modifications/network-modification-menu.type';
+import MenuItem from '@mui/material/MenuItem';
+import { UUID } from 'crypto';
+import { setCurrentRootNetworkUuid } from 'redux/actions';
+import ListItemText from '@mui/material/ListItemText';
+
+export interface AppBreadCrumbsProps {
+    studyName: string;
+    studyPath: string;
+}
```

Comment:
```suggestion
export interface BreadcrumbsProps {
    studyName: string;
    studyPath: string;
}
```
and add just above component